lib/mp4/probe.js

```diff
@@ -87,52 +88,55 @@ timescale = function(init) {
  * fragment, in seconds
  */
 startTime = function(timescale, fragment) {
-  var trafs, baseTimes, result;
+  var trafs, result;
```

Comment:
This code was all kinds of weird. We create an empty array twice to map over another array of trafs?? Then we map over all tfhd/tfdt and parse them, but only take the first? It's all kinds of nuts. 

I switched it so that we loop over all `traf`s, and only grab the lowest time that we find (which is what we ultimately did later in the code). Furthermore we only look for one `tfhd` and one `tfdt` as there will only ever be one, and we ignored other in the old code anyway.